pages/doc/tracing_best_practices.md

```diff
@@ -108,11 +106,7 @@ The Wavefront service uses these names as span tag values, as filters for traces
 
 ### Best Practices for Wavefront Observability SDKs
 
-* Set up an [Application tags object](trace_data_details.html#application-tags) in each service to define logical names for the application constructs.
-  - Specify the logical application and service names that apply to the service. Optionally include logical cluster and shard names, if you want to use the physical topology to filter your data.
-  - Define a custom tag called `component` if you are using the Wavefront OpenTracing SDK. <br>
-
-    {% include tip.html content="Other Wavefront tracing SDKs automatically define `component` for you. " %}
+* Set up an [Application tags object](trace_data_details.html#application-tags) in each service to define logical names for the application constructs. Specify the logical application and service names that apply to the service. Optionally include logical cluster and shard names, if you want to use the physical topology to filter your data.
```

Comment:
... Optionally, if you want to use the physical topology to filter your data, include logical cluster and shard names.